src/main/java/Operation.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+
+public class Operation {
+
+    public int runOperation(String input) {
+        String[] values = input.split(" ");
+        int result = 0;
+        String operator = "";
+
+        try {
+            checkValidation(values);
+            result = Integer.parseInt(values[0]);
+            result = getResult(values, result, operator);
```

Comment:
```suggestion
            result = getResult(values, Integer.parseInt(values[0]), operator);
```
코드는 언제나 짧은게 좋습니당

Reply:
```suggestion
            result = getResult(values, operator);
```
`getResult` 메소드에서 values를 파라미터로 받으니, 내부에서 0번째 값만 꺼낼 수도 있겠네요?